ilastik/workflows/carving/preprocessingDrawer.ui

```diff
@@ -26,7 +26,7 @@
    <string>PreprocessDrawer</string>
   </property>
   <property name="locale">
-   <locale language="German" country="Germany"/>
+   <locale language="English" country="UnitedStates"/>
```

Comment:
I don't know if it wouldn't be better to remove the locale tag alltogether, since we don't use it anywhere else and just let the system decide how to show stuff.